### ISSUE TITLE
generate data file with cariage return

### ISSUE THREAD

**Issue by mantoze** [phalcon/phalcon-devtools#631]
If data in longtext field with carriage return, then command
`phalcon migration generate --data=oncreate`
generate data file with carriage returns.
When run "phalcon migration run" i have error:

```
1458228958.5245: INSERT INTO `contents` (
    `id`,
    `unique_id`,
    `slug_id`,
    `article_title`,
    `article`,
    `short_description`,
    `itinerary`,
    `created_at`,
    `updated_at`
)
VALUES (?, ?, ?, ?, ?)

ERROR: SQLSTATE[21S01]: Insert value list does not match column list:
1136 Column count doesn't match value count at row 1
```

I think need change in Migration.php. Example:

``` php
$data[] = "'".addslashes(str_replace(array("\n", "\n\r", "\r"), '', $value))."'";
```

**Comment by lenrsmith** [comment]
@sergeyklay  I believe PR #919 fixes this problem on the 3.0.x branch.

**Comment by sergeyklay** [comment]
Yes, thanks!
